AssertionError in test_get_path_distance()

Using https://github.com/onnela-lab/forest/tree/hackdna/python38 with Python 3.8:

```
    def test_get_path_distance(coords1, coords2, directions1, mocker):
        mocker.patch(
            "openrouteservice.Client.directions", return_value=directions1
        )
>       assert (
            get_path(coords1, coords2, Vehicle.CAR, "mock_api_key")[1]
            == 843.0532531565476
        )
E       assert 843.0532478112679 == 843.0532531565476

forest/bonsai/tests/test_simulate_gps_data.py:283: AssertionError
```

https://github.com/onnela-lab/forest/runs/4875764517

This error does not occur when I run the test suite locally on macOS Big Sur with Python 3.8.

--- a/forest/bonsai/tests/test_simulate_gps_data.py
+++ b/forest/bonsai/tests/test_simulate_gps_data.py
@@ -281,8 +281,8 @@ def test_get_path_distance(coords1, coords2, directions1, mocker):
         "openrouteservice.Client.directions", return_value=directions1
     )
     assert (
-        get_path(coords1, coords2, Vehicle.CAR, "mock_api_key")[1]
-        == 843.0532531565476
+        round(get_path(coords1, coords2, Vehicle.CAR, "mock_api_key")[1], 2)
+        == 843.05
     )
 
 

